optaplanner-core/src/main/java/org/optaplanner/core/config/heuristic/selector/entity/pillar/PillarSelectorConfig.java

```diff
@@ -26,17 +26,22 @@
 import org.optaplanner.core.config.heuristic.selector.common.SelectionCacheType;
 import org.optaplanner.core.config.heuristic.selector.common.SelectionOrder;
 import org.optaplanner.core.config.heuristic.selector.entity.EntitySelectorConfig;
+import org.optaplanner.core.config.heuristic.selector.move.generic.PillarType;
 import org.optaplanner.core.config.util.ConfigUtils;
 import org.optaplanner.core.impl.domain.variable.descriptor.GenuineVariableDescriptor;
 import org.optaplanner.core.impl.heuristic.selector.entity.EntitySelector;
 import org.optaplanner.core.impl.heuristic.selector.entity.pillar.DefaultPillarSelector;
 import org.optaplanner.core.impl.heuristic.selector.entity.pillar.PillarSelector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import static org.apache.commons.lang3.ObjectUtils.*;
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
 
 @XStreamAlias("pillarSelector")
 public class PillarSelectorConfig extends SelectorConfig<PillarSelectorConfig> {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(PillarSelectorConfig.class);
```

Comment:
From ScoreDirectorFactoryConfig.java:

    private static final Logger logger = LoggerFactory.getLogger(ScoreDirectorFactoryConfig.class);

As it is a "static final", it makes sense to make it uppercase according to typical java guidelines - but I'd like that change to be done consistently for all optaplanner-core code - in a separate PR I guess.

Do or do not - when we change something in optaplanner-core, we either do it consistently everywhere or we don't do it at all. I don't want to build car that has modern rubber wheels on the left side and still wooden wheels on the right side.